src/containers/sb-file-uploader.jsx

```diff
@@ -107,7 +128,11 @@ class SBFileUploader extends React.Component {
                         nonInteraction: true
                     });
                     // Remove the hash if any (without triggering a hash change event or a reload)
-                    history.replaceState({}, document.title, '.');
+                    try { // Can fail e.g. when GUI is loaded from static file (integration tests)
```

Comment:
interesting

Reply:
Yeah I only realized it when I saw that the integration test was failing because it was hitting the "catch" alert condition below.